client/src/app.jsx

```diff
@@ -1,19 +1,167 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { Title } from '../style.js';
+import ItemList from './components/itemList.jsx';
+import axios from 'axios';
 
 class App extends React.Component {
   constructor (props) {
     super (props);
     this.state = {
-      item: null
+      ids: [],
+      photos: [],
+      names: [],
+      prices: [],
+      brands: [],
+      ratings: [],
+      starRatings: []
     };
+    this.getSimilarItems = this.getSimilarItems.bind(this);
+    this.getItemDescription = this.getItemDescription.bind(this);
+    this.getItemNames = this.getItemNames.bind(this);
+    this.getItemBrands = this.getItemBrands.bind(this);
+    this.getPrices = this.getPrices.bind(this);
+    this.getPhotos = this.getPhotos.bind(this);
+    this.getRatings = this.getRatings.bind(this);
+  }
+
+  //get item info
+  getItemDescription (productIds) {
+    let item1 = productIds[0];
+    let item2 = productIds[1];
+    let item3 = productIds[2];
+    let item4 = productIds[3];
+    let item5 = productIds[4];
+    let item6 = productIds[5];
+    axios.get(`http://ec2-18-217-85-161.us-east-2.compute.amazonaws.com:4004/descriptions/multiple?${item1}=${item1}&${item2}=${item2}&${item3}=${item3}&${item4}=${item4}&${item5}=${item5}&${item6}=${item6}`)
+      .then((response) => {
+        let itemDescriptions = response.data;
+        this.getItemNames(itemDescriptions);
+        this.getItemBrands(itemDescriptions);
+        this.getPrices();
+        this.getPhotos();
+        this.getRatings();
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  }
+
+  //get ItemNames
+  getItemNames (items) {
+    let itemNames = [];
+    for (let i = 0; i < items.length; i++) {
+      itemNames.push(items[i].itemName);
+    }
+    this.setState({
+      names: itemNames
+    });
+  }
+  //get Item Brands
+  getItemBrands (items) {
+    let itemBrands = [];
+    for (let i = 0; i < items.length; i++) {
+      itemBrands.push(items[i].brand);
+    }
+    this.setState({
+      brands: itemBrands
+    });
+  }
+
+  //get price
+  getPrices () {
+    let multipleIds = { ids: this.state.ids};
+    axios.post('http://localhost:4003/priceandinventory/id/multiple', multipleIds.ids)
```

Comment:
shouldn't your `post` requests be `get` requests?

Reply:
I believe it should be but Alek had set it up to receive post request when requesting multiple items.